web/src/stream_ui_updates.js

```diff
@@ -103,11 +79,11 @@ export function update_settings_button_for_sub(sub) {
     }
     if (stream_data.can_toggle_subscription(sub)) {
         $settings_button.prop("disabled", false);
-        $settings_button.popover("destroy");
+        $settings_button.parent()[0]._tippy?.destroy();
```

Comment:
We've observed in production that `$settings_button.parent()[0]` may be undefined, leading to exceptions from this line of code.

Reply:
Thanks for sharing the update. :heart_decoration:  I will look into this one and see why it is happening.

Reply:
@amanagr filed #25841 for this, so I believe this is fixed.